packages/glibc.rb

```diff
@@ -3,169 +3,259 @@
       aarch64: '64b4b73e2096998fd1a0a0e7d18472ef977aebb2f1cad83d99c77e164cb6a1d6',
        armv7l: '64b4b73e2096998fd1a0a0e7d18472ef977aebb2f1cad83d99c77e164cb6a1d6',
-       x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997',
+       x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997'
+    })
+  when '2.32' # All architectures with updates past M92.
+    source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.32.tar.xz'
+    source_sha256 '1627ea54f5a1a8467032563393e0901077626dc66f37f10ee6363bb722222836'
+
+    binary_url({
+      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.32_x86_64/glibc-2.32-chromeos-x86_64.tpxz'
+    })
+    binary_sha256({
+      x86_64: '0e6749f4d4c445e27bcd8669865fa060193552ecda2233f7f0d5e91c90b91a0b'
     })
   end
 
   depends_on 'gawk' => :build
+  depends_on 'hashpipe' => :build
   depends_on 'libgd' => :build
+  depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
 
   def self.patch
-    # Apply patch due to new version of binutils which causes compilation failure
-    # http://lists.busybox.net/pipermail/buildroot/2017-August/199812.html
-    Dir.chdir "misc" do
-      if File.readlines("regexp.c").grep(/monitor/).any?
-        puts "Patched!".lightgreen
-      else
-        system "sed -i 's,char \\*loc1,char \\*loc1 __attribute__ ((nocommon)),g' regexp.c"
-        system "sed -i 's,char \\*loc2,char \\*loc2 __attribute__ ((nocommon)),g' regexp.c"
-        system "sed -i 's,char \\*locs,char \\*locs __attribute__ ((nocommon)),g' regexp.c"
+    case LIBC_VERSION
+    when '2.23', '2.27'
+      # Apply patch due to new version of binutils which causes compilation failure
+      # http://lists.busybox.net/pipermail/buildroot/2017-August/199812.html
+      Dir.chdir 'misc' do
+        unless File.readlines('regexp.c').grep(/monitor/).any?
+          system "sed -i 's,char \\*loc1,char \\*loc1 __attribute__ ((nocommon)),g' regexp.c"
+          system "sed -i 's,char \\*loc2,char \\*loc2 __attribute__ ((nocommon)),g' regexp.c"
+          system "sed -i 's,char \\*locs,char \\*locs __attribute__ ((nocommon)),g' regexp.c"
+          puts 'Patched!'.lightgreen
+        end
+      end
+    when '2.32'
+      FileUtils.mkdir_p 'gentoopatches'
+      system 'curl -Ls https://dev.gentoo.org/~dilfridge/distfiles/glibc-2.32-patches-8.tar.xz | \
+        hashpipe sha256 6653f1d0aadad10bd288f3bae274bd4e0a013d47f09ce78199fb5989b2d8fd9b | \
+        tar -xJf - -C gentoopatches'
+      Dir.glob('gentoopatches/patches/*.patch').each do |patch|
+        system "patch -Np1 -i #{patch}"
+      end
+      @googlesource_branch = 'release-R92-13982.B'
+      system "git clone --depth=1 -b  #{@googlesource_branch} https://chromium.googlesource.com/chromiumos/overlays/chromiumos-overlay googlesource"
+      Dir.glob('googlesource/sys-libs/glibc/files/local/glibc-2.32/*.patch').each do |patch|
+        puts patch
+        system "patch -Np1 < #{patch}"
```

Comment:
Use glob?
`patch -Np1 < googlesource/sys-libs/glibc/files/local/glibc-2.32/*.patch`

Reply:
Does that work?